frontend/app/src/components/dashboard/LiquidityProviderBalanceTable.vue

```diff
@@ -0,0 +1,325 @@
+<template>
+  <dashboard-expandable-table v-if="balances.length > 0">
+    <template #title>
+      {{ $t('dashboard.liquidity_provider.title') }}
+      <v-btn :to="route" icon class="ml-2">
+        <v-icon>mdi-chevron-right</v-icon>
+      </v-btn>
+    </template>
+    <template #details>
+      <v-menu
+        id="nft_balance_table__column-filter"
+        transition="slide-y-transition"
+        max-width="250px"
+        offset-y
+      >
+        <template #activator="{ on }">
+          <menu-tooltip-button
+            :tooltip="$t('dashboard_asset_table.select_visible_columns')"
+            class-name="ml-4 nft_balance_table__column-filter__button"
+            :on-menu="on"
+          >
+            <v-icon>mdi-dots-vertical</v-icon>
+          </menu-tooltip-button>
+        </template>
+        <visible-columns-selector
+          group="LIQUIDITY_PROVIDER"
+          :group-label="$t('dashboard.liquidity_provider.short').toString()"
+        />
+      </v-menu>
+    </template>
+    <template #shortDetails>
+      <amount-display
+        :value="total"
+        show-currency="symbol"
+        fiat-currency="USD"
+      />
+    </template>
+    <data-table
+      :headers="mainHeaders"
+      :items="balances"
+      item-key="nftId"
```

Comment:
In what kind of format is the id? numerical or a string?

Reply:
string